src/egon/data/datasets/heat_timeseries_etrago_table/hts_etrago.py

```diff
@@ -0,0 +1,119 @@
+            # interlinking bus_id and area_id
+            bus_area = db.select_dataframe(
+                 f"""
+                 SELECT bus_id, area_id, id FROM
+                 {targets['heat_buses']['schema']}.
+                 {targets['heat_buses']['table']}
+                 JOIN {sources['district_heating_areas']['schema']}.
+                     {sources['district_heating_areas']['table']}
+                 ON ST_Transform(ST_Centroid(geom_polygon), 4326) = geom
+                 WHERE carrier = '{carrier}'
+                 AND scenario='{scenario}'
+                 """,
+                 index_col="id",
+             )
+            
+            #district heating time series time series
+            disct_time_series = db.select_dataframe(
+                                """
+                                SELECT * FROM 
+                                demand.egon_timeseries_district_heating
+                                """
+                                )
+            ## bus_id connected to corresponding time series
+            bus_ts = pd.merge(bus_area,disct_time_series, on='area_id', how = 'inner')
+        
+        else:
+            #interlinking bus_id and subst_id
+            bus_sub = db.select_dataframe(
+                 f"""
+                 SELECT bus_id, subst_id FROM
+                 {targets['heat_buses']['schema']}.
+                 {targets['heat_buses']['table']}
+                 JOIN {sources['egon_mv_grid_district']['schema']}.
+                             {sources['egon_mv_grid_district']['table']}
+                 ON ST_Transform(ST_Centroid({sources['egon_mv_grid_district']['schema']}.
+                             {sources['egon_mv_grid_district']['table']}.geom), 
+                                 4326) = {targets['heat_buses']['schema']}.
+                                         {targets['heat_buses']['table']}.geom
+                 WHERE carrier = '{carrier}'
+                 """
+             )
+            ##scenario name still needs to be adjusted in bus_sub
+            
+            #individual heating time series
+            ind_time_series = db.select_dataframe(
+                                """
+                                SELECT * FROM 
+                                demand.egon_etrago_timeseries_individual_heating
+                                """
+                                )
+            
+            ## bus_id connected to corresponding time series
+            bus_ts = pd.merge(bus_sub,ind_time_series, on='subst_id', how = 'inner')
+              
+        next_id = next_etrago_id('load')
+        
+        bus_ts['load_id']=np.arange(len(bus_ts))+next_id
+        
+        etrago_load = pd.DataFrame(index=range(len(bus_ts)))
+        etrago_load['scn_name'] = scenario
+        etrago_load['load_id'] = bus_ts.load_id
+        etrago_load['bus'] =bus_ts.bus_id
+        etrago_load['type'] =None  ### where are these values comming from?##########*#*#*
```

Comment:
The column values `None` or `NaN` are the default values. As you already wrote in the comment it is a bit confusing were they are coming from. So I suggest to simply delete these columns from your dataframe.